### PR TITLE
Update examples and docs to work with latest pdfmake

### PR DESCRIPTION
**Pull request by rob-mccann**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -495,20 +495,18 @@ Example `.babelrc` file:
 {
   "presets": [
     [
-      "env",
+      "@babel/preset-env",
       {
         "targets": {
-          "node": "6"
+          "node": "current"
```
Comment (mtlewis) [inline README.md:501]: If we end up supporting 14 on the other PR, I guess this should be updated too:

```suggestion
          "node": "14.0”
```

**Review by mtlewis — APPROVED** [review]
Awesome!

File: README.md (hunk 1 above)
Comment (rob-mccann) [reply README.md:501]: fyi this is the README - I set this to `current` to leave the `engines` in `package.json` to indicate support.

Thought I'd leave this as current to save the chore of updating it when there's a new nodejs release - that OK?

Comment (mtlewis) [reply README.md:501]: Makes total sense, I missed that I was looking at the readme 😁

**Pull request merged**